recipes/date/all/CMakeLists.txt

```
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 2.8.11)
+cmake_minimum_required(VERSION 2.8.12)
```

How come the hooks didn't complain about this one?
```suggestion
cmake_minimum_required(VERSION 3.1)
```

---

Please check recent discussion with @Croydon. Hook KB-H048 does not complaint about 2.x if cxx standard is not specified.

---

Thanks, I took a look at the implementation of a hook and see what it is trying to check.

https://github.com/conan-io/hooks/blob/5d3c51a97e1341000dc62643a04556f78ec8fe35/hooks/conan-center.py#L488